cpp/src/server/serverChat.cc

```diff
@@ -36,7 +37,8 @@ int serverChat(int sockfd) {
       if (!FD_ISSET(currentClientFd, &master)) continue;
 
       std::fill_n(buffer, BUFFER_SIZE, '\0');
-      const int bytesRead = recv(currentClientFd, buffer, sizeof(buffer), 0);
+      // sizeof(buffer) - 1 to avoid garbage values
+      const int bytesRead = recv(currentClientFd, buffer, sizeof(buffer)-1, 0);
```

Comment:
no data is lost due to this 1 byte? please confirm

Reply:
No data loss, confirmed